### PR TITLE
Doc onDevTokenSignInHandlers

### PR DESCRIPTION
**Pull request by akanex115**
# Background
https://github.com/ajktown/docs/issues/42

## TODOs
- [x] Doc `useDevTokenSignInHandlers()`

## Checklist (Developers)
- [x] Make this PR as a draft first
- [x] Title is checked
- [x] Background & TODOs are filled
- [x] Assignee is set
- [x] Labels are set
- [x] Project is created & linked, if multiple PRs are associated to this PR
- [x] Appropriate issue(s) is(are) linked to this PR under `development`
- [x] `yarn inspect` is run
- [x] `TODOs` are handled and checked
- [x] Final Operation Check is done
- [x] Make this PR as an open PR

## Checklist (Reviewers)
- [x] Check if there are any other missing TODOs that are not yet listed
- [x] Review Code
- [x] Every item on the checklist has been addressed accordingly
- [x] If `development` is associated to this PR, you must check if every TODOs are handled

### CODE REVIEW

**Review by mlajkim — COMMENTED** [review]
first review

File: dev/diagrams/on-dev-token-sign-in-handlers.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      activate api
+        hook <- api: Successful authentication with a developer token
+        hook -> authPrep: onGetAuthPrep()
+        activate authPrep
+          authPrep -> api: onGetAuthPrep
+          break Returns null
+            authPrep <- api: Return null
+          end break
+          authPrep <- api: Return data
+        deactivate api
```
[the remark this answers is not in the document]
Comment (akanex115) [reply dev/diagrams/on-dev-token-sign-in-handlers.md]: Handled in [2b522c2](https://github.com/ajktown/docs/pull/48/commits/2b522c2bad669cb802613cb6322870b91655f957)

File: dev/diagrams/on-dev-token-sign-in-handlers.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    activate hook
+      hook -> api: postAuthByDevTokenApi()
+      activate api
+        hook <- api: Successful authentication with a developer token
+        hook -> authPrep: onGetAuthPrep()
+        activate authPrep
+          authPrep -> api: onGetAuthPrep
+          break Returns null
+            authPrep <- api: Return null
+          end break
```
[the remark this answers is not in the document]
Comment (akanex115) [reply dev/diagrams/on-dev-token-sign-in-handlers.md]: Handled in[ae92c1f](https://github.com/ajktown/docs/pull/48/commits/ae92c1f6db7507600d953671d02e55b2e838d293)

File: dev/diagrams/on-dev-token-sign-in-handlers.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        deactivate api
+        hook <- authPrep: Return undefined
+      deactivate authPrep
+      hook <- hook: router.push(DEFAULT_MAIN_APP_PAGE)
+      fe <- hook: Returns nothing
+      break if an error occurs
+        hook <- hook: throw new Error(`something went wrong`)
+        hook <- hook: onError console.log(`onError; ContinueWithDevToken`)
+        fe <- hook: Returns nothing
+      end break
```
[the remark this answers is not in the document]
Comment (akanex115) [reply dev/diagrams/on-dev-token-sign-in-handlers.md]: Handled in [98e5031](https://github.com/ajktown/docs/pull/48/commits/98e50319552b915f4f734590fa9affd9bb335fb5)

[the remark this answers is not in the document]
Comment (mlajkim) [reply dev/diagrams/on-dev-token-sign-in-handlers.md]: ok

**Review by mlajkim — APPROVED** [review]
OK

**Pull request merged**